src/current/cockroachcloud/cloud-api.md

```diff
@@ -380,10 +420,66 @@ Where:
+  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
   {{site.data.alerts.end}}
   - `{secret_key}` is the secret key for the service account.
-  - `{spend_limit}` is the [maximum amount of money, in US cents, you want to spend per month]({% link cockroachcloud/plan-your-cluster.md %}) on this cluster.
+  - `storage_mib_limit` is the maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled, where only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this value to be zero.
+  - `request_unit_limit` is the maximum number of request units that the cluster can consume during the month. If this limit is exceeded, then the cluster is disabled until the limit is increased, or until the beginning of the next month when more request units are granted. It is an error for this to be zero.
+
+If the request was successful, the client will not receive a response payload.
+
+{{site.data.alerts.callout_info}}
+The previously deprecated `spend_limit` field has been removed. Use the `usage_limits` field instead to set resource limits for the cluster.
+{{site.data.alerts.end}}
+
+## Change provisioned vCPUs for a Standard cluster
+
+To update the provisioned vCPUs for a cluster, send a `PATCH` request to the `/v1/clusters/{cluster_id}` endpoint.
+
+{{site.data.alerts.callout_success}}
+The service account associated with the secret key must have the Cluster Administrator or Cluster Developer [role]({% link cockroachcloud/authorization.md %}#organization-user-roles).
+{{site.data.alerts.end}}
+
+The service account associated with the secret key must have `ADMIN` or `EDIT` [permission]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) to retrieve information about an organization's clusters.
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="curl"><strong>curl</strong></button>
+    <button class="filter-button page-level" data-scope="raw"><strong>Raw</strong></button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="curl">
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request PATCH \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id} \
+  --header 'Authorization: Bearer {secret_key}' \
+  --data '{"usage_limits": {"provisioned_virtual_cpus": "{provisioned_virtual_cpus}"}}'
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="raw">
+
+{% include_cached copy-clipboard.html %}
+~~~ json
+{
+  "usage_limits": {
+    "provisioned_virtual_cpus": "{provisioned_virtual_cpus}"
+  }
+}
+~~~
+
+</section>
+
+Where:
+
+  - `{cluster_id}` is the unique ID of this cluster.
```

Comment:
If you take the suggestions above about switching to separating with a `:`, I suppose these should be kept in sync. I wonder if there is potential for reuse on the common ones (like cluster ID) by saving them into Liquid variables.

Reply:
saved as follow up task